windowsdefenderatp_connector.py

```diff
@@ -1447,14 +1447,31 @@ def _handle_list_sessions(self, param):
 
         return action_result.set_status(phantom.APP_SUCCESS)
 
-    def _handle_prevalence(self, param):
+    def _handle_ip_prevalence(self, param):
+        action_identifier = self.get_action_identifier()
+        self.save_progress("In action handler for {}".format(action_identifier))
+        self.debug_print("Calling _handle_prevalence method")
```

Comment:
does just putting these in the shared handler not count for the minimum logging check? iinm, logs in the shared function call should still be counted per call as opposed to once

Reply:
I could be wrong, but I don't think so because the error messages from last time stated that the handler functions couldn't be found (they didn't conform to the standard usage.)

Reply:
oh yeah, that would be addressed by the "handler wrappers" you added, but I meant the check isn't only checking the literal inside of the handler body. I'm pretty sure it also goes into any called function bodies as well

Reply:
Ok, well nobody told me that. Is there a place where these rules are documented?

Reply:
hmm, I don't think we do. or at least I'm not aware of anything beyond just the test code and its documentation: https://cd.splunkdev.com/phantom/app_release/-/blob/master/app_tests/code_tests.py#L530 which also isn't currently visible for external contributors for that matter

cc: @pzhou-splunk